src/Compiler.php

```diff
@@ -246,6 +256,9 @@ private function handleTwigConfig(string $twigConfig): void
             $attributes = array_map(function ($item) { return trim($item); }, $attributes);
             $this->attributesWithIf = array_merge($this->attributesWithIf, $attributes);
         }
+        if ($config['disable-data-support'] ?? false) {
```

Comment:
Vue to Twig turned it off by default, because I think it's a risky change.
I want to activate it in the bundle.
The logic here allows deactivating for a single vue file.